backend/infrastructures/apps/load/views/views.py

```diff
@@ -0,0 +1,97 @@
+import logging
+from pathlib import Path
+from typing import cast
+
+import inject
+from pandera.typing.pandas import DataFrame
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from modules.common.domain import exceptions as domain_exceptions
+from modules.data.domain import value_objects as data_value_objects
+from modules.extract.domain import commands as domain_extract_commands
+from modules.extract.services import commands as service_extract_commands
+from modules.load.domain import commands as domain_load_commands
+from modules.load.domain.ports import units_of_work
+from modules.load.services import commands as services_load_commands
+from modules.load.services.queries import ports as query_ports
+from modules.transform.domain import commands as domain_transform_commands
+from modules.transform.services import commands as services_transform_commands
+
+from .serializers import ExtractDataSerializer, OutputDataSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class DataViewSet(
+    ViewSet,
+):
+    @inject.param(name="save_data_unit_of_work", cls="save_data_unit_of_work")
+    def create(
```

Comment:
This MUST be moved to `extarct` Django app.

Reply:
done in new PR